src/Pages/yeheum/Login/Login.js

```diff
@@ -30,28 +27,37 @@ class Login extends React.Component {
     this.props.history.push("/main-yeheum");
   };
 
-  postToLocal = () => {
-    fetch("", {
-      method: "GET",
-      headers: {
-        Authorization: token,
-      },
+  handleClickPost = () => {
+    const { email, pw } = this.state;
+    fetch("http://10.58.4.237:8000/user/signin", {
+      method: "POST",
       body: JSON.stringify({
-        id: "kim",
-        password: "1234",
+        email: email,
+        password: pw,
       }),
     })
       .then((response) => response.json())
       .then((response) => {
-        if (response.token) {
-          localStorage.setItem("wtw-token", response.token);
+        console.log(response);
+        if (response.message === "WRONG_PASSWORD") {
+          this.setState({
+            errorMessage: "비밀번호가 일치하지 않습니다.",
+          });
+        } else if (response.message === "INVALID_USER") {
+          this.setState({
+            errorMessage: "아이디가 없습니다.",
+          });
+        } else if (response.message === "Register_Success") {
+          localStorage.setItem("Authorization", response.access_token);
+          this.goToMain();
```

Comment:
- console.log는 테스트 끝났으면 제때 삭제하고 push 해주세요.
- 함수의 파라미터 자리에서 직접 구조분해할당 해줄 수 있습니다.
```suggestion
      .then(({ message, access_token }) => {
        const msgTable = {
          "WRONG_PASSWORD": "비밀번호가 일치하지 않습니다.",
          "INVALID_USER": "아이디가 없습니다."
        }
        
        if (message === "REGISTER_SUCCESS") {
          localStorage.setItem("Authorization", access_token);
          this.goToMain();
          return;
        }
        
        this.setState({ errorMessage: msgTable[message] })  
```
- register_success만 대문자로 작성하지 않은 이유가 있나요?
- setState는 최대한 마지막에 한번만 해주는 쪽으로 작성해주세요
- 일반적인 통신 상황에서는 에러 핸들링을 try/catch 문을 사용하게 됩니다.